src/App.tsx

```diff
@@ -276,51 +217,6 @@ function App(): ReactElement {
     setupInitialParameters();
   }, [isInitialDatasetLoaded]);
 
-  // Add event listeners for unloading and resizing on startup.
-  useEffect(() => {
-    window.addEventListener("beforeunload", () => {
-      canv.domElement.removeEventListener("click", handleCanvasClick);
-      canv.dispose();
-    });
```

Comment:
Removing this because the browser is likely taking care of this, and the `beforeunload` event is unreliable in certain browser contexts (https://developer.chrome.com/articles/page-lifecycle-api/#the-beforeunload-event)

Reply:
These event listeners that hold on to references to things like canv, and would prevent canv from ever being garbage collected.   In this case, I think we generally expect the listener and the canvas to last for the whole browser session and won't get re-created over and over, right?

Reply:
Yeah, I can't see a situation where we'd need to remake the canvas.